clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -387,6 +387,17 @@ public class VenicePushJob implements AutoCloseable {
   private static final String TEMP_DIR_PREFIX = "/tmp/veniceRmdSchemas/";
   public static final int NOT_SET = -1;
   private static final Logger LOGGER = LogManager.getLogger(VenicePushJob.class);
+  /**
+   * Config to enable single-colo/region canary push for VPJ.
+   * In this mode, the VPJ will only push data to the source colo/region first, perform validation and finally
+   * leverage data recovery/repush to propagate data globally.
+   * The source colo is determined by Controller using the given priority order:
+   *     1. Parent controller emergency source fabric config.
+   *     2. VPJ plugin source grid fabric config, aka {@link #SOURCE_GRID_FABRIC}.
+   *     3. Store level source fabric config.
```

Comment:
you mean `nativeReplicationSourceFabric` store config or you are planning to add new one?

Reply:
I mean `nativeReplicationSourceFabric` store config.